Gemfile

```diff
@@ -9,7 +9,7 @@ gem 'coffee-rails', '~> 4.2'
 gem 'jquery-rails'# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
-gem 'bootstrap-sass'
+gem 'bootstrap-sass', '~> 3.2.0'
```

Comment:
This makes your code more portable, as you need bootstrap 3 to run it. On systems that have an older version installed, your app will see that bootstrap is already installed but then will hit errors when it runs.